common/services/batch-request.js

```diff
@@ -29,7 +31,7 @@ const batchRequests = async (clientMethodCall, props = {}, propPaths = []) => {
       chunks.map(chunk => {
         const batchedProps = cloneDeep(props)
         set(batchedProps, propPath, chunk)
-        return clientMethodCall(batchedProps)
+        return clientMethodCall({ ...batchedProps, apiClient: apiClient })
```

Comment:
Happy to be told off for this, but essentially, we don't want to use the cloned api client (mocks don't work, for example). We want to use the same client that was passed in.

Reply:
This whole method smells :(

Reply:
I am hoping the comment at the beginning of it about it being able to be removed once the services know the user's locations is true and we can do that in the new year.